src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSemanticFactsService.vb

```diff
@@ -255,5 +255,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return DirectCast(n.GetSyntax(cancellationToken), TypeStatementSyntax).Modifiers.Any(SyntaxKind.PartialKeyword)
                 End Function)
         End Function
+
+        Public Function IsPossibleTupleContext(semanticModel As SemanticModel, position As Integer, cancellationToken As CancellationToken) As Boolean Implements ISemanticFactsService.IsPossibleTupleContext
+            Throw New NotImplementedException()
```

Comment:
I had to add it to `ISemanticFactsService` which this class implements, and the C# implementation is used, but the VB implementation would be un-used at this point.